SU-SOE/soe_basic
# D8CORE-3983: fixing the teasers to  have the SOE link style

# READY FOR REVIEW

# Summary
- Fixing the teasers to get the SOE link styling

# Review By (Date)
- 3/26

# Urgency
- Med

# Steps to Test

1. Pull in this change.
2. Clear caches.
3. Add a news and people teasers.
4. make sure SOE links look right.

# Affected Projects or Products
- SOE theming on teasers

# Associated Issues and/or People
- D8CORE-3983
- 
# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
